docs/advanced/index.md

```diff
@@ -6,7 +6,7 @@ Most users won’t typically need to interact with the library but instead use a
 
 ## Usage
 
-For advanced use cases you can use the library directly.  By default, MDX is asynchronous because plugins can be asynchronous themselves.  This means that plugins can request data, read from the file system.  Anything!
+For advanced use cases you can use the library directly.  By default, MDX is asynchronous because plugins can be asynchronous themselves!  This means that plugins can make request data, read from the file system.  Anything!
```

Comment:
This change reads a bit weirdly, was this the intent?

```suggestion
For advanced use cases you can use the library directly.  By default, MDX is asynchronous because plugins can be asynchronous themselves!  This means that plugins can make requests for data, read from the file system.  Anything!
```

Reply:
Oh, whoops. Definitely a typo. Nice catch!